### PR TITLE
Tf 1.13

### PR DESCRIPTION
**Pull request by dfalbel**
This is a WIP PR to a 1.13.1 CRAN release.

Tested against:

- [X] MacOS - TF 1.12.0
- [x] MacOS - TF 1.13.1
- [x] Windows - TF 1.12.0
- [x] Windows - TF - 1.13.1
- [x] Travis - TF 1.13.1

### CODE REVIEW

File: R/generics.R [hunk 1]
```diff
@@ -288,26 +288,26 @@
...
-Arg.tensorflow.tensor <- function(x) {
-  tf$angle(x)
+Arg.tensorflow.tensor <- function(z) {
+  tf$angle(z)
 }
 
 #' @export
-Mod.tensorflow.tensor <- function(x) {
-  tf$abs(x)
+Mod.tensorflow.tensor <- function(z) {
```
Comment (terrytangyuan) [inline R/generics.R]: Just curious - why are these changes needed?

Comment (dfalbel) [reply R/generics.R]: This deals with those CRAN warnings about methods consistency. It seems that the methods for complex numbers are defined in terms of `z`. 

```
* checking S3 generic/method consistency ... WARNING
Arg:
  function(z)
Arg.tensorflow.tensor:
  function(x)

Conj:
  function(z)
Conj.tensorflow.tensor:
  function(x)

Im:
  function(z)
Im.tensorflow.tensor:
  function(x)

Mod:
  function(z)
Mod.tensorflow.tensor:
  function(x)

Re:
  function(z)
Re.tensorflow.tensor:
  function(x)
```
